pulsar-common/src/main/java/org/apache/pulsar/common/conf/InternalConfigurationData.java

```diff
@@ -28,19 +28,19 @@
 
     private String zookeeperServers;
     private String configurationStoreServers;
-    private String ledgersRootPath;
+    private String metadataServiceUri;
```

Comment:
It seems `ledgersRootPath` is much more direct than `metadataServiceUri`. metadata may lead developer think of "zookeeper". How about change this back？

Reply:
@jiazhai In fact, `metadataServiceUri` here is a uri to connect to a zookeeper cluster, which is used to find bookies. I would prefer to keep it since `ledgersRootPath` seems more like a path and is using together with zookeeper. As sijie's comment above, we are trying to use a metadata service uri, so how about `bkMetadataServiceUri`?